### ISSUE TITLE
Please release a new version

### ISSUE THREAD

**Issue by gabebw** [thoughtbot/stylelint-config#77]
The latest version (4.0.0, installed by Suspenders) shows this warning, which is already fixed in the `main` branch:

```
Deprecation warnings:
 - The "scss/at-import-partial-extension-blacklist" rule is deprecated.
 - 'at-import-partial-extension-blacklist' has been deprecated, and will be removed in '7.0'. Use 'at-import-partial-extension-disallowed-list' instead. See: https://github.com/stylelint-scss/stylelint-scss/blob/v6.1.0/src/rules/at-import-partial-extension-blacklist/README.md
```

**Comment by nickcharlton** [comment]
I've been investigating what I thought might be a regression in BEM support, at least, I was before I went on holiday. I'll see how I get on with that this week, worst case I'll cut a release on Friday.
